snippets/csharp/System.Net/WebClient/Overview/useragent.cs

```diff
@@ -19,11 +19,11 @@ public static void Main(string[] args)
         client.Headers.Add("user-agent", "Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.2; .NET CLR 1.0.3705;)");
 
         using Stream data = client.OpenRead(args[0]);
-        StreamReader reader = new StreamReader(data);
+        using StreamReader reader = new StreamReader(data);
         string s = reader.ReadToEnd();
         Console.WriteLine(s);
         data.Close();
         reader.Close();
```

Comment:
Do we still need the explicit `Close()` calls?

Reply:
Removed